templates/reconciliation.js

```diff
@@ -35,6 +35,48 @@ var vote = function vote($choice){
   nextPairing($pairing);
 }
 
+// This function looks through allVotes() to see if there are any IDs
+// matched with multiple UUIDs or (conversely) any UUIDs matched with
+// multiple IDs. If so, it returns HTML that warns about these
```

Comment:
Oh, I certainly meant that the interface would be a new thing — just trying to make sure that that can be purely additive, rather than having to also split things like this up.